source/templates/partials/_modal-test.twig

```diff
@@ -6,4 +6,23 @@
   btnReset: 'Annuler'
 } %}
 
-{% block modalContent %}This is modal content{% endblock %}
+{% block formContent %}
+
+<div class="form__wrapper">
+  <label class="form__label" for=""></label>
+  <input class="form__field" id="" type="text" name="" value="">
+</div>
+
+<div class="form__wrapper">
+  <label class="form__label" for=""></label>
+  <input class="form__field" id="" type="date" name="" value="">
```

Comment:
For date inputs, in the UI there is a text about the format "Format JJ/MM/AAAA", we need to add it to this file for accessibility (we need to tell users what format of data they need to write.

- Add a `<p>` tag next to your input with format text
- We need to "connect" input with this text format for screen readers, we can do it with `aria-describedby` attribute (https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute), try to implement it (read docs)